<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMQualiResultWidget</class>
 <widget class="QWidget" name="QMQualiResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>697</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>10</number>
     </property>
     <widget class="QDockWidget" name="dwFilter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>366</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>524287</height>
       </size>
      </property>
      <property name="floating">
       <bool>false</bool>
      </property>
      <property name="features">
       <set>QDockWidget::NoDockWidgetFeatures</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::AllDockWidgetAreas</set>
      </property>
      <property name="windowTitle">
       <string>Filter</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QComboBox" name="cbFilterEmployeeGroup">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Schulung</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QToolButton" name="tbResetFunc">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Zurücksetzen</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rcs/icons.qrc">
            <normaloff>:/icons/icons/sc_undo.png</normaloff>:/icons/icons/sc_undo.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QToolButton" name="tbResetTrain">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Zurücksetzen</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rcs/icons.qrc">
            <normaloff>:/icons/icons/sc_undo.png</normaloff>:/icons/icons/sc_undo.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="cbFilterEmployee">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QComboBox" name="cbTrainResultState">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>Gut</string>
          </property>
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
          <item>
           <property name="text">
            <string>Gut</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ausreichend</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Schlecht</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="tbExtSelEmployeeGroup">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Erweiterte Auswahl</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rcs/icons.qrc">
            <normaloff>:/icons/icons/lc_selectdb.png</normaloff>:/icons/icons/lc_selectdb.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="3">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Ergebnisstatus</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QToolButton" name="tbExtSelTrainResultState">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Erweiterte Auswahl</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rcs/icons.qrc">
            <normaloff>:/icons/icons/lc_selectdb.png</normaloff>:/icons/icons/lc_selectdb.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QComboBox" name="cbFilterTrain">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QToolButton" name="tbResetTrainResultState">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Zurücksetzen</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rcs/icons.qrc">
            <normaloff>:/icons/icons/sc_undo.png</normaloff>:/icons/icons/sc_undo.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QToolButton" name="tbResetEmployeeGroup">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Zurücksetzen</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rcs/icons.qrc">
            <normaloff>:/icons/icons/sc_undo.png</normaloff>:/icons/icons/sc_undo.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Funktion</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Mitarbeitergruppe</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="tbExtSelEmployee">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Erweiterte Auswahl</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rcs/icons.qrc">
            <normaloff>:/icons/icons/lc_selectdb.png</normaloff>:/icons/icons/lc_selectdb.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Mitarbeiter</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="tbResetEmployee">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Zurücksetzen</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../rcs/icons.qrc">
            <normaloff>:/icons/icons/sc_undo.png</normaloff>:/icons/icons/sc_undo.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="tbFilterReset">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Zurücksetzen</string>
            </property>
            <property name="icon">
             <iconset resource="../rcs/icons.qrc">
              <normaloff>:/icons/icons/sc_undo.png</normaloff>:/icons/icons/sc_undo.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tbUpdate">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Aktualisieren</string>
            </property>
            <property name="icon">
             <iconset resource="../rcs/icons.qrc">
              <normaloff>:/icons/icons/sc_reload.png</normaloff>:/icons/icons/sc_reload.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="12" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <widget class="QComboBox" name="cbFilterFunc">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="tbFilterVisible">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Filtern</string>
           </property>
           <property name="icon">
            <iconset resource="../rcs/icons.qrc">
             <normaloff>:/icons/icons/lc_goalseekdialog.png</normaloff>:/icons/icons/lc_goalseekdialog.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbCreateReport">
           <property name="text">
            <string>Bericht</string>
           </property>
           <property name="icon">
            <iconset resource="../rcs/icons.qrc">
             <normaloff>:/icons/icons/sc_reportnavigator.png</normaloff>:/icons/icons/sc_reportnavigator.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="tbSaveToCsv">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Export CSV</string>
           </property>
           <property name="icon">
            <iconset resource="../rcs/icons.qrc">
             <normaloff>:/icons/icons/text-csv.svg</normaloff>:/icons/icons/text-csv.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="tbSaveToPDF">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Export PDF</string>
           </property>
           <property name="icon">
            <iconset resource="../rcs/icons.qrc">
             <normaloff>:/icons/icons/lc_exportdirecttopdf.png</normaloff>:/icons/icons/lc_exportdirecttopdf.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2">
         <item>
          <widget class="QTableView" name="tvQualiResult">
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>200</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>25</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>25</number>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rcs/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dwFilter</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>filterVisibilityChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbResetEmployee</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>resetEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbResetEmployeeGroup</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>resetEmployeeGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbResetFunc</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>resetFunc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbResetTrain</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>resetTrain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbResetTrainResultState</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>resetTrainResultState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbExtSelEmployee</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>extSelEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbExtSelEmployeeGroup</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>extSelEmployeeGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbExtSelTrainResultState</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>extSelTrainResultState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbSaveToCsv</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>saveToCsv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>907</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbSaveToPDF</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>printToPDF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1013</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFilterReset</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>resetFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFilterVisible</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>switchFilterVisibility()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbUpdate</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiResultWidget</receiver>
   <slot>updateFilterAndCalculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateFilterAndCalculate()</slot>
  <slot>switchFilterVisibility()</slot>
  <slot>printToPDF()</slot>
  <slot>resetFilter()</slot>
  <slot>saveToCsv()</slot>
  <slot>filterVisibilityChanged()</slot>
  <slot>resetEmployee()</slot>
  <slot>resetFunc()</slot>
  <slot>resetTrain()</slot>
  <slot>resetEmployeeGroup()</slot>
  <slot>resetTrainResultState()</slot>
  <slot>extSelEmployee()</slot>
  <slot>extSelEmployeeGroup()</slot>
  <slot>extSelTrainResultState()</slot>
 </slots>
</ui>
